src/components/Search/Searchbar.jsx

```diff
@@ -1,10 +1,32 @@
 import React from 'react';
 
-const SearchBar = ({}) => (
-  <form className="Search-bar">
-    <input type="text"></input>
-    <input type="submit" value="Search"></input>
-  </form>
-);
+class SearchBar extends React.Component {
```

Comment:
On all other components, I imported Component as "import React, { Component } from 'react'" and then "class SearchBar extends Component". Can we change this for consistency?